.github/workflows/narwhal_pull.yml

```diff
@@ -0,0 +1,40 @@
+name: Update Dependencies
+on:
+  ## Allow triggering this workflow manually via GitHub CLI/web
+  workflow_dispatch:
+  schedule:
+    # Update on every hour at 10 past the hour
+    - cron:  '10 * * * *'
+
+jobs:
+  update-dep:
+    runs-on: ubuntu-latest
+    # Important settings as we don't want to open a PR when the update fails
+    continue-on-error: false
```

Comment:
When an update will cause build failure, does the failure get detected by PR CI or inside the workflow?